articles/discipline-flow.md

```diff
@@ -0,0 +1,62 @@
+
+Both roles, the collaboration owner and the project manager (PM) are not part of the sketch above, since both are often more concerned with the complete flow and not so much with discipline-specifics directly, at least in their roles CO and PM. The PM of course often also considers the ROI of the product in relation to the project's budget and hence is part of the BUS discipline. Similar the CO might be part of a particular discipline. More importantly from flow perspective, both roles together ensure that not only features are delivered, but that they are also delivered in time and in budget with appropriate quality by a team that sees purpose individually as well as on level of the product development as a whole.
+
+## Successful Products
+Even if we have achieved project excellence it is not ensured that we have delivered a *successful* product. These require additionally that user and business needs and goals are explicitly taken into account from early on. We need to develop and deliver incrementally and incorporate feedback and learnings continuously. Each discipline has its specific contribution:
+
+- BIZ: models with statements about value and impact
+- CX: overarching product(s) delivery and experience concept
+- UX: statements about product value and specific concepts for user needs
+- REQ: concrete requirements and metrics for the product development
+- ARCH: feasable technical context for development and production
+- IMPL: adaptable, maintanable and operatable implementation of what is needed
+- OPS: concrete environments, pipelines and monitoring for development and production
+
+One key aspect to achieve product excellence is to track and monitor product features (or increments) as they make their way through the complete cyclical flow in order to learn and improve continuously within and from each discipline. That is all the way from CX/UX to delivery in production systems, including the tracking and monitoring of the product's usage and business performance.
+
+## Conclusion
+In general the Discipline Flow makes the big picture of disciplines and their relations a first class citizen and allows to explain what we sometimes call 'Operational Excellence'. This kind of excellence is a result of discipline experts working together within a lean project organisation for product delivery that focuses on user and market needs from the very beginning on. The flow comprises only a small number of disciplines and relations, simple enough to sketch it ad-hoc whenenver needed for discussions and complete enough to check the continuous delivery maturity in context of your current project. 
```

Comment:
I am not sure what the Discipline Flow is and what I should do with it. 
Can you explain that in german maybe and I help to translate?

Reply:
Es ist im Prinzip eine Konzeptskizze für das Zusammenwirken der verschiedenen Disziplinen im Software Engineering, oder allgemeiner in der (digitalen) Produktentwicklung. 

Ein paar Use-Case Beispiele sind der Längenrestriktion zum Opfer gefallen ;-) Die sind hier auf meiner Wiki-Seite: https://confluence.zuehlke.com/display/~stj/Discipline+Flow#DisciplineFlow-SpecificUseCasesfortheModel

Reply:
Sollen wir die Use-Cases wieder reinnehmen? Weil spätestens nach der Conclusion sollte Dir schon ein bisschen klar sein "what the Discipline Flow is and what I should do with it." :) Wenn nicht, dann ist der Artikel noch nicht fertig ;-)

Reply:
Ich würde ihn wider reinnehmen.

Reply:
ok, mach ich